---
name: Jenna
surname: Hamlin
position: "Bioinformatician II"
address: "Respiratory Diseases Laboratory Branch, Centers for Disease Control and Prevention, Atlanta, GA, USA"
email: "jhamlin@cdc.gov"
github: jennahamlin
aboutme: "I am a scientist with 10+ years of experience in classical genetics, genomics, and evolutionary biology. My training includes using a range of tools - bioinformatics, population genetics, molecular evolution all with experimental design in mind. Over the years as my computational skills increased, the genomes of the organisms I work on have gotten smaller. Now, I work at the interface of public health, application development, and understanding organisms that can cause harm to humans with an empahsis of developing reproduciable research."
date: "`r format(Sys.time(), '%B %Y')`"
headcolor: 414141
output: vitae::awesomecv
--- 


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
library(here)
library(magrittr)
library(glue)
```

```{r get data, include=FALSE}

## Education
edu <- readr::read_csv(here::here("data", "education.csv"))

## Appointments
appointments <- readr::read_csv(here::here("data", "appointments.csv"))
 
## Skills
 skills <- readr::read_csv(here::here("data", "skills.csv")) %>% 
   dplyr::group_by(category) %>% 
   dplyr::mutate(skill = paste(skill, collapse = ", ")) %>% 
   dplyr::ungroup()%>% 
   unique
 
## Publications - my google id here
pubs<-scholar::get_publications("OIIf5T4AAAAJ&hl")%>%
  dplyr::mutate(author = author%>%
                as.character %>%
                stringr::str_trim(),
                first_author = dplyr::case_when(stringr::str_starts(author, "JAP Hamlin") ~ TRUE,
                                                TRUE ~ FALSE),
                preprint = dplyr::case_when(journal %in% c("bioRxiv", "BioRxiv") ~ TRUE,
                                            TRUE ~ FALSE)) %>%

  dplyr::filter(journal  != "University of Georgia" & journal != "Bulletin of the Ecological Society of America") %>% #exclude thesis
  dplyr::arrange(dplyr::desc(year))

pubs$journal <-plyr::revalue(pubs$journal,c("G" = "G3")) #necessary as when pubs imported from scholar G3 becomes G, this corrects that

## Presentations - talks
presentations <- readr::read_csv(here::here("data", "presentations.csv")) %>% 
  dplyr::arrange(desc(year))

## Software
packages <- readr::read_csv(here::here("data", "packages.csv"))

## Mentoring
mentor <- readr::read_csv(here::here("data", "mentor.csv"))

```

# Education

```{r education}
vitae::detailed_entries(edu, 
                 when = awarded,
                 what = degree,
                 where = loc,
                 with = uni)

```

# Appointments

```{r appointments}


## adjusted code so that I code have sub-bullet point explaination
# appointments %>% 
#   vitae::detailed_entries(
#     what = role,
#     when = glue("{start_date}--{end_date}"),
#     with = company,
#     where = where,
#     why = details)

## should move code to seperate R script and source it, because as is this is messy
tibble::tribble(
  ~ role, ~ company, ~ when, ~ where, ~ details,
  "Bioinformatician II with Division of Bacterial Diseases, Salary - 95,000 (USD)",
  "Centers for Disease Control and Prevention,  Contractor with ASRT Inc.",
  "November 2020 - Current",  "Atlanta, GA",  
  "[] * Interfacing with state public health laboratories (PHL)
   \\begin{itemize}
     \\item Communicated with members of state PHL to provide guidenance regarding bionformatics and genomics for Legionella species
   \\end{itemize}
   \\vspace{1mm}
   * Refactored Legionella Species ID tool for broader compatibility and interfacing with Nextflow Tower
   \\begin{itemize}
     \\item Removed structural design issues in original code that prevented expansion and adding of new functionalities 
   \\end{itemize}
   \\vspace{1mm}
   * Development of method validation plan for Legionella Species ID tool
   \\begin{itemize}
     \\item Researched and proposed statically sound validation plan for tool that uses next generation sequencing data
   \\end{itemize}
   \\vspace{1mm}
   * Phylogenetic and phylodyanamic analyses for Legionella isolates of public health concern
   \\begin{itemize}
      \\item Performing analyses and generating statistical summaries and visualizations
      \\item Writing and editing manuscripts for publication
      \\item Identifying scientific questions answerable with existing data
   \\end{itemize}
    \\vspace{1mm}
   * Developing or testing scripts and pipelines as necessary
   \\begin{itemize}
      \\item Generated script to interact with National Center of Biotechnology Information for downloading all available Legionella genomes
      \\item Evaluated using Nextflow pipeline for mapping Legionella genomes
   \\end{itemize}
    \\vspace{1mm}
  * 60-day detail on Data Analytics and Visualization for Vaccine Task Force (April - June 2021)
   \\begin{itemize}
     \\item Produced weekly reports and consolidated steps for document creation to be reproducible via one script 
   \\end{itemize}
   \\vspace{1mm}",
  "Association of Public Health Laboratories (APHL) Bioinformatics Fellow",
  "Centers for Disease Control and Prevention; Fellow with APHL",
  "September 2019 - November 2020",  "Atlanta, GA",  
  "[] * Developing tinselR - an Rshiny application for annotating outbreak trees
   \\begin{itemize}
     \\item tinselR is an open-source downloadable package in the R language used for phylogenetic tree analysis and visualization
   \\end{itemize}
   \\vspace{1mm}
   * Participated in APHL core competency and leadership training
   \\begin{itemize}
     \\item Gained skills and knowledge for a career in public health  
   \\end{itemize}
   \\vspace{1mm}
   * 30-day detail on Routine Reporting for COVID-19 Response (June - July 2020)
   \\begin{itemize}
     \\item Produced daily and weekly reports for task force. Consolidated steps for document creation to be reproducible via one script 
   \\end{itemize}",
  "Bensasson Lab Post-Doctoral Researcher",
  "University of Georgia (UGA)",
  "October 2017 - August 2019",  "Athens, GA",  
  "[] * Directed collaborative research with UGA graduate and undergraduate students \\newline
     \\vspace{1mm} 
     * Collated and streamlined genome mapping for 1000 Saccharomyces cerevisiae isolates as lab resource \\newline
     \\vspace{1mm} 
     * Used iterative approach to assign 400 genomes to well-defined populations using best practices for population genomics and phylogenetics \\newline
     \\vspace{1mm} 
     * Resolved haplotypes for Candida albicans genomes with PacBio sequencing",
  "Moyle Lab Post-Doctoral Researcher",
  "Indiana University (IU)",
  "June 2015 - September 2017",  "Bloomington, IN",  
    "[] * Directed research with faculty, graduate, and undergraduates at IU and with two external universities \\newline
     \\vspace{1mm} 
     * Genome-wide study of factors that determine the likelihood of introgression for Solanum species \\newline
     \\vspace{1mm} 
     * Streamlined genotyping to generate biological replicates. Validate data accuracy using technical replicates along with statistical analyses \\newline
     \\vspace{1mm} 
     * Identified targeting rates and floral trait differences with biological \\& technical replicates in large-scale crossing project via statistical analyses") %>%
  vitae::detailed_entries(
    what = role,
    when = when, #glue::glue("{start_date}--{end_date}"),
    with = company,
    where = where,
    why = details, .protect = FALSE)

```

# Skills

```{r skills}
skills %>% 
  vitae::detailed_entries(what = skill, with = category)
```

# Publications

```{r publications}
pubs %>%
  dplyr::filter(!preprint) %>%
  vitae::detailed_entries(
    what = title,
    when = year,
    with = author,
    where = journal,
    .protect = TRUE
  )
```

# Presentations

**Talks**

```{r talks}
presentations %>%
  dplyr::filter(type == "talk")%>%
  vitae::detailed_entries(
    what = venue,
    when = year,
    with = title,
    where = city,
    .protect = TRUE)
```

**Posters**

```{r posters}
presentations %>%
  dplyr::filter(type == "poster")%>%
  vitae::detailed_entries(
    what = venue,
    when = year,
    with = title,
    where = city,
    .protect = TRUE)
```

# R - Packages

```{r packages}

packages %>% 
  vitae::detailed_entries(
    what = role,
    when = year,
    with = title,
     where = location,
    .protect = TRUE)
```

# Mentoring

**University of Georgia graduate students**

```{r uga grad}
mentor %>%
   dplyr::filter(where == "University of Georgia" & level == "graduate")%>%
  vitae::brief_entries(
    what = who)
```

**University of Georgia undergraduate students**

```{r uga undergrad}
mentor %>%
   dplyr::filter(where == "University of Georgia" & level == "undergraduate")%>%
  vitae::brief_entries(
    what = who)
```

**Indiana University graduate students**

```{r iu grad}
mentor %>%
   dplyr::filter(where == "Indiana University" & level == "graduate")%>%
  vitae::brief_entries(
    what = who)
```

**Indiana University undergraduate students**

```{r iu undergrad}
mentor %>%
   dplyr::filter(where == "Indiana University" & level == "undergraduate")%>%
  vitae::brief_entries(
    what = who)
```
